item-spec/item-spec.md

```diff
@@ -165,6 +165,7 @@ or streamed. It is allowed to add additional fields.
 | description | string    | A description of the Asset providing additional details, such as how it was processed or created. [CommonMark 0.29](http://commonmark.org/) syntax MAY be used for rich text representation. |
 | type        | string    | [Media type](#media-types) of the asset. |
 | roles       | \[string] | The [semantic roles](#asset-role-types) of the asset, similar to the use of `rel` in links. |
+| *Item Property* | varies | Any Item property can be defined for each asset |
```

Comment:
I think I'd remove this row and replace it as follows:

```suggestion

It is allowed to add **additional fields**:
* [STAC Common Metadata](common-metadata.md#stac-common-metadata) - A list of fields commonly used throughout all domains.
* [Content Extensions](../extensions/README.md#list-of-content-extensions) - Domain-specific fields 
such as EO, SAR and point clouds.
* [Custom Extensions](../extensions/README.md#extending-stac) - It is generally allowed to add custom 
fields. [See above for more information.](#additional-fields).
```

Also, remove the "It is allowed to add additional fields." above.

Reply:
I wasn't quite sure how to present it, I like the suggestion, will update